web/src/locales/en/data.json

```diff
@@ -1254,5 +1256,12 @@
     "Single org only": "Single org only",
     "Single org only - Tooltip": "Triggered only in the organization that the webhook belongs to",
     "Value": "Value"
+  },
+  "widgets": {
```

Comment:
Update i18n in this way: https://casdoor.org/docs/contributing#how-to-update-i18n-data